es/cheatsheet-machine-learning-tips-and-tricks.md

```diff
@@ -1,257 +1,257 @@
 
 <br>
 
 **18. [Total sum of squares, Explained sum of squares, Residual sum of squares]**
 
-&#10230;
+&#10230; [Suma total de los cuadrados, Explicaicón de la suma de cuadrados, Residio de la suma de cuadrados]
 
-<br>
+<br> 
 
 **19. Coefficient of determination ― The coefficient of determination, often noted R2 or r2, provides a measure of how well the observed outcomes are replicated by the model and is defined as follows:**
 
-&#10230;
+&#10230; Coefficientes de determinante ― El coeficiente de determinante, usualmente llamado R2 o r2, provee una medida 
 
 <br>
 
 **20. Main metrics ― The following metrics are commonly used to assess the performance of regression models, by taking into account the number of variables n that they take into consideration:**
 
-&#10230;
+&#10230; Métricas principales ― Las siguientes métricas son comunmente usadas para evaluar el desempeño de los modelos de regresión, tomando el número de variables n que son consideradas:
 
 <br>
 
 **21. where L is the likelihood and ˆσ2 is an estimate of the variance associated with each response.**
 
-&#10230;
+&#10230; donde L es la posibilidad y ˆσ2 es una estimación de varianza asociada a cada respuesta.
 
 <br>
 
 **22. Model selection**
 
-&#10230;
+&#10230; Selección de modelo
 
 <br>
 
 **23. Vocabulary ― When selecting a model, we distinguish 3 different parts of the data that we have as follows:**
 
-&#10230;
+&#10230; Vocabulario ― Cuando se selección un modelo, se tienen 3 diferentes partes de la información como se muestra:
 
 <br>
 
 **24. [Training set, Validation set, Testing set]**
 
-&#10230;
+&#10230; [conjunto de entrenamiento, conjunto de validación, conjunto de prueba]
 
 <br>
 
 **25. [Model is trained, Model is assessed, Model gives predictions]**
 
-&#10230;
+&#10230; [Entrenamiento de modelo, Evaluación de modelo, Predicción]
 
 <br>
 
 **26. [Usually 80% of the dataset, Usually 20% of the dataset]**
 
-&#10230;
+&#10230; [Usualmente 80% del dataset, Usualmente 20% del dataset]
 
 <br>
 
 **27. [Also called hold-out or development set, Unseen data]**
 
-&#10230;
+&#10230; [También llamado resistencia o conjunto de desarrollo, información no antes vista]
 
 <br>
 
 **28. Once the model has been chosen, it is trained on the entire dataset and tested on the unseen test set. These are represented in the figure below:**
 
-&#10230;
+&#10230; Una vez que el modelo se ha escogido, se entrena sobre todo el dataset y se prueba en información no antes vista. Esto se representa en la figura siguiente:
 
 <br>
 
 **29. Cross-validation ― Cross-validation, also noted CV, is a method that is used to select a model that does not rely too much on the initial training set. The different types are summed up in the table below:**
 
-&#10230;
+&#10230; Validación cruzada ― También nombrada CV, es un método que se usa para seleccionar un modelo que no se basa demasaido en el test de entrenamiento inicial. Los diferentes tipos estan englobados en la siguiente tabla:
```

Comment:
* You misspelt _demasiado_.
* "does not rely too much on the initial training set" -> _no se basa demasiado en el conjunto de entrenamiento inicial_. Where did you add the word "test"?